apps/ecommerce/providers/shopify/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{
+  "include": ["src", "test"],
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
```

Comment:
Do we want to keep all these comments in here?

Reply:
Don't have a strong opinion here. Preferentially, I personally like having all these options. Also I placed all the configuration we are using to the top of the file